x-pack/test/scalability/config.ts

```diff
@@ -62,6 +69,29 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ...(!!AGGS_SHARD_DELAY ? ['--data.search.aggs.shardDelay.enabled=true'] : []),
         ...(!!DISABLE_PLUGINS ? ['--plugins.initialize=false'] : []),
       ],
+      env: {
+        ELASTIC_APM_ACTIVE: true,
+        ELASTIC_APM_CENTRAL_CONFIG: false,
+        ELASTIC_APM_TRANSACTION_SAMPLE_RATE: '0.1',
```

Comment:
nit: is this intentional that we'll only get 10% of the APM data?

Reply:
It will introduce extra load by capturing all requests, but wouldn't it be better to be more conservative in our estimates and try to capture more data?

I don't feel that strongly about it, but if the tradeoff for getting more accurate data is that we are stressing the system a bit more than we would under ordinary circumstances, it seems like that might be acceptable.

Reply:
My opinion:

These tests by definition create hundreds of requests per second over a sustained period. We don't need to know the APM traces of every requests that was made to get representative results. Because these tests are standardised (same request body, same sample data etc) the spans between the lowest 1% and the 99% would be identical except for some variation in the timing. Unless Elasticsearch is the bottleneck the timing variation would come from the event loop being blocked and APM doesn't show you details of this anyway. So you open a span and there's an unexplained 5s block of open space. More samples from APM doesn't help us understand what causes the white block.

Unless we can confirm that the 100% sampling rate doesn't affect performance significantly I think we should have a good theory for when this extra data would be useful to justify skewing our results.